src/prof.c

```diff
@@ -310,6 +310,7 @@ prof_leave(tsd_t *tsd, prof_tdata_t *tdata) {
 }
 
 #ifdef JEMALLOC_PROF_LIBUNWIND
+JEMALLOC_ALIGNED(CACHELINE)
```

Comment:
Whoops I just verified again and it's not needed. I somehow got the idea about non-aligned functions trigger slow path in libunwind. Turns out it's triggered by stack misalignment instead of function. I'll submit a diff removing this.